### ISSUE TITLE
Sync from multiple PCs

### ISSUE THREAD

**Issue by pefp2008** [HearthSim/Hearthstone-Deck-Tracker#1742]
Hi.

I have 2 PCs (a laptop and a desktop), both with HS and HSDT.
Last week, HSDT wasn't syncing because hearthstats was down, but i still played on both computers during that week, using HSDT to keep my records.

Now, that hearthstats is working properly i found that both HSDTs have different stats between each other, even if i have been syncing them at startup, and before i close HS and HSDT.

Is this how its supposed to work or its an error?

**Comment by Cu8eR** [comment]
Try DropBox next time. Anyway it should work fine when you sync your all matches - every match has own time and date.

Try force full sync

**Comment by lexblow** [comment]
I have the same problem here. Using a desktop at work, and desktop at home. Forced full sync on the work desktop and it imported all decks, but  with no stats at all. All appeared as NO STATS. How can i fix this so i have a trully full sync with stats and all? ty!

**Comment by YanTS** [comment]
Hi.

I have the same issue. I was using HDT with Hearthstats sync, and sync settings were set as most rigorous (sync always, everything).
Now I wiped my system and did a clean win10 install, downloaded HDT and synced with Hearthstats. Decks downloaded ok from the site, but matches (games) not - Stats window is empty in HDT, but when I go to the site, I can see all those games I played yesterday.
Particularily constructed matches.

The behavior I want to report: Hearthstats sync does not download matches from Hearthstats.

This also leads to the "No stats" message on every deck.
